### PR TITLE
fixing dockerfile, the nipype_tutorial dir was read-only

### PR DESCRIPTION
**Pull request by djarecka**

### CODE REVIEW

**Comment by miykael** [comment]
Thanks for the changes. I was curious if we should add again the notebooks from
https://github.com/neuro-data-science/neuroviz.git
https://github.com/neuro-data-science/neuroML.git
https://github.com/ReproNim/reproducible-imaging.git

What do you think? It might be not relevant for nipype itself but for anybody who wants to learn neuroimaging with python (which is a partial goal of this tutorial, for me).

**Comment by djarecka** [comment]
what about adding links to them somewhere (in e.g. external resources with some short advertisement)? It might be just hard to follow changes in those repositories.
